cmd/osctl/cmd/cluster.go

```diff
@@ -383,7 +382,7 @@ func parseCPUShare() (int64, error) {
 }
 
 func init() {
-	clusterUpCmd.Flags().StringVar(&nodeImage, "image", "docker.io/autonomy/talos:"+version.Tag, "the image to use")
+	clusterUpCmd.Flags().StringVar(&nodeImage, "image", defaultImage(), "the image to use")
```

Comment:
I guess this should be different from `--install-image`?

Reply:
Thanks for catching.